test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,9 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  test("returns string of names, seperated by commas and an ampersand", () => {
```

Comment:
Again, similar to what is mentioned [here](https://github.com/mike-shields-dev/command-shift-diy-kata-code-reviews/pull/4/commits/6385ca8530ed4882477a33c335c2720cd6c7c3ec#r1258115593), consider adding more assertions to the test to add more confidence. 

Something else to consider is how does the `joinNames` function behave when passed an array of name objects with a length < 3 ? :thinking: